cogs/allCommands.py

```diff
@@ -12,58 +12,63 @@ def __init__(self, bot):
         embed=discord.Embed(title = "***" + messagelist[0] + "***")
-        numbers_to_words = ['0️⃣', '1️⃣', '2️⃣', '3️⃣', '4️⃣', '5️⃣', '6️⃣', '7️⃣', '8️⃣', '9️⃣', '🔟']
+        numbers_to_words = ['1️⃣', '2️⃣', '3️⃣', '4️⃣', '5️⃣', '6️⃣', '7️⃣', '8️⃣', '9️⃣', '🔟']
         for i in range(len(messagelist)):
-            if i > 0:
-                embed.add_field(name = "", value = numbers_to_words[i] + " " + messagelist[i], inline=False)
+            if i > 1:
+                embed.add_field(name = "", value = numbers_to_words[i - 2] + " " + messagelist[i], inline=False)
         msg = await ctx.send(embed=embed)
         for j in range(len(messagelist)):
-            if j > 0:
-                await msg.add_reaction(numbers_to_words[j])
-
-        await sleep(60) #Allow one minute for voting phase, plan to add input for user to determine voting time
+            if j > 1:
+                await msg.add_reaction(numbers_to_words[j - 2])
 
-        #---LOADING PHASE---#
-        #await ctx.channel.purge(limit=1) <----- May add this back, may want to clean up as to not see the old poll after results post
-        embed=discord.Embed(title = "***" + "Poll results final!" + "***")
-        embed.add_field(name = "", value = "Poll: " + messagelist[0], inline=False)
-        embed.add_field(name = "", value = "Here are the results:", inline=False)
-        embed.add_field(name = "", value = "Loading...", inline=False)
-        temp = await ctx.send(embed=embed)
-
-        await sleep(3) #Allow short pause to allow bot to retrieve reactions on poll
+        await sleep(int(messagelist[1]) * 60) #Voting phase length is now determined by user input
 
         #---RESULTS PHASE---#
         refreshed_msg = await ctx.fetch_message(msg.id)
         reactions_array = refreshed_msg.reactions
-        await ctx.channel.purge(limit=1)
+        #await ctx.channel.purge(limit=1) May add this back, may want to clean up as to not see the old poll after results post
         embed=discord.Embed(title = "***" + "Poll results final!" + "***")
         embed.add_field(name = "", value = "Poll: " + messagelist[0], inline=False)
         embed.add_field(name = "", value = "Here are the results:", inline=False)
         for entry in range(len(reactions_array)):
-            embed.add_field(name = "", value = reactions_array[entry].emoji + " " + messagelist[entry + 1] + " --> " + str(reactions_array[entry].count - 1), inline=False)
+            embed.add_field(name = "", value = reactions_array[entry].emoji + " " + messagelist[entry + 2] + " --> " + str(reactions_array[entry].count - 1), inline=False)
         results_msg = await ctx.send(embed=embed)
+
+    @commands.command(
+        help="Allows users to delete a specified amount of messages in the current chat (default = 1). Format: '$purge [amount]'",
+        brief="Format: $purge [amount]"
+    )
+    async def purge(self, ctx, amount=1):
+        await ctx.channel.purge(limit=amount+1)
```

Comment:
We definitely need to add a confirmation upon calling the purge command and then make it permissions based but this was a rlly good PR, especially for the voting fixes